shared/redux/rtk-apis/classrooms/classrooms.api.ts

```diff
@@ -1,16 +1,27 @@
 import { TClassroomDetailsDto } from "@/modules/UserDashboard/components/ClassroomCreatingModal/ClassroomCreatingModal.types";
 
 import projectApi from "../api.config";
-import { TClassroomApiResponse } from "./classrooms.types";
+import { TStudentInfoApiResponse } from "../users/users.types";
+import { TClassroomApiResponse, TClassroomsApiResponse } from "./classrooms.types";
 
 const classroomsApi = projectApi.injectEndpoints({
   endpoints: (builder) => ({
-    getClassrooms: builder.query<TClassroomApiResponse, void>({
+    getClassroomById: builder.query<TClassroomApiResponse, string>({
+      query: (id) => `classrooms/${id}`,
+      providesTags: ["Classroom"],
+    }),
+
+    getEnrolledStudents: builder.query<TStudentInfoApiResponse, string>({
+      query: (id) => `classrooms/students/${id}`,
```

Comment:
```suggestion
      query: (id) => `classrooms/${id}/students`,
```
Please follow REST architecture